internal/cmd/controller/grutil/status.go

```diff
@@ -99,6 +107,64 @@ func SetStatusFromBundleDeployments(ctx context.Context, c client.Client, gitrep
 	return nil
 }
 
+// SetStatusFromGitjob sets the status fields relative to the given job in the gitRepo
+func SetStatusFromGitjob(ctx context.Context, c client.Client, gitRepo *fleet.GitRepo, job *batchv1.Job) error {
```

Comment:
This function's body seems to mostly come from `updateStatus` in `gitjob_controller.go`, which is no longer called and should then be deleted.

Reply:
yup, deleted. I wonder why `golang-ci` didn't complain... Maybe we should enable more linters